### DISCUSSION TITLE
dynamic video-crop

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#13904] [Q&A]
Hi, I want to setup the horizontal video-crop 2pixels below the actuall video height. So when the video is 3840x2100 the crop is set to 3840x2098. Is this possible?

Thank you :)

**Comment by kasper93** [comment]
I don't think it is currently possible. You can do `--video-crop=100%x99%+0+0` which would crop 1% off the height from the bottom, but absolute negative values are not supported. Maybe it could be added.

**Reply by geextahslex** [reply]
Is it possible to go higher than 99% like 99.5 ?
